src/selection.js

```diff
@@ -202,10 +199,7 @@ class Selection {
 
     _updatedTouchingElements() {
         const touched = [];
-        const changed = {
-            added: [],
-            removed: []
-        };
+        const changed = {added: [], removed: []};
```

Comment:
lol, sorry, this was unnecessary; couldn't resist and ended up committing.

Reply:
Hahaha, no prob... such short objects could be written in one line :)